packages/wonder-blocks-toolbar/components/toolbar.md

```diff
@@ -60,10 +73,15 @@ const tryAgain = <Button kind="secondary">Try again</Button>;
 const {default: Icon, icons} = require("@khanacademy/wonder-blocks-icon");
 const IconButton = require("@khanacademy/wonder-blocks-icon-button").default;
 const Link = require("@khanacademy/wonder-blocks-link").default;
+const {LabelLarge} = require("@khanacademy/wonder-blocks-typography");
 
 const closeButton = <IconButton icon={icons.dismiss} kind="tertiary" style={{ width: 28 }} />;
```

Comment:
I think the style here should be `style={{marginLeft: -12}}`

Reply:
Sure, sounds good. I wasn't sure about the focus ring brushing up against the border like that when testing before.